src/main/java/fshop/service/workfile/WriteCsvImpl.java

```diff
@@ -0,0 +1,40 @@
+package fshop.service.workfile;
+
+import fshop.db.FoodDao;
+import fshop.model.Food;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+public class WriteCsvImpl implements WriteCsv {
+    private FoodDao foodDao;
+    private String fileName;
+
+    public WriteCsvImpl(String fileName) {
+        Objects.requireNonNull(fileName);
+        this.fileName = fileName;
+    }
+
+    @Override
+    public Map<Food, Integer> write() {
+        Map<Food, Integer> mapFromDb = foodDao.getDataOfBalance();
```

Comment:
same here. Do this operation in main method and pass this map to write method along with fileName